rclpy/rclpy/node.py

```diff
@@ -98,7 +99,7 @@ def __init__(
             services.
         :param initial_parameters: A list of parameters to be set during node creation.
         """
-        self._handle = None
+        self.__handle = None
```

Comment:
Is this rename intentionally to break code using it directly? Or is it only changed to match the same attribute in other classes?

Reply:
The latter. I used `__handle` in `Subscription` so I changed it here too.